packages/main/src/ColorPalette.js

```diff
@@ -238,6 +236,19 @@ class ColorPalette extends UI5Element {
 		dialog.open();
 	}
 
+	/**
+	 * Returns the selected color.
+	 *
+	 * @readonly
+	 * @type { String }
+	 * @defaultvalue undefined
+	 * @public
+	 * @since 1.0.0-rc.15
+	 */
+	get selectedColor() {
```

Comment:
Should we call this getter ```value```?

Reply:
Initially I called it "value", because the property used to be "value', then I thought the ColorPalette has `selectedColor` rather than a `value`. Somehow I see `value` related to the ColorPaletteItem.

Reply:
IMO, selectedColor gives more information, but if we want consistency we will have to stick to 'value' name. Is there an option, where we can rename the _value private property to _selectedColor for example? This will give us more information.